src/modules/TodoItem.js

```diff
@@ -0,0 +1,7 @@
+export default class TodoItem {
+  constructor(data) {
+    this.index = data.index;
+    this.description = data.description;
+    this.completed = data.completed;
+  }
+}
```

Comment:
- [ ] [Optional] Please consider the fusion of this class with the TodoList class as it's just creating an object and that's not the idea of using classes. I understand your wish of working with more than one class but to keep the real purpose of classes please kindly make this change.